### PR TITLE
Created models & migrated database

### PR DESCRIPTION
**Pull request by annelaws**
- Models created
- Devise added to User
- DB migrated & Schema created

### CODE REVIEW

**Comment by bkneussl** [comment]
great!!

**Comment by bkneussl** [comment]
good work! :)

**Comment by lauwhite** [comment]
ok for me as well

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ gem 'sassc-rails'
 gem 'simple_form'
 gem 'uglifier'
 gem 'webpacker'
```
Comment (bkneussl) [inline Gemfile]: oh no why do we need this?

File: Gemfile [hunk 2]
```diff
@@ -14,6 +14,7 @@ gem 'sassc-rails'
...
+gem 'devise'
```
Comment (summerlin1995) [inline Gemfile]: nice gem :)

File: Gemfile [hunk 3]
```diff
@@ -14,6 +14,7 @@ gem 'sassc-rails'
...
```
Comment (lauwhite) [inline Gemfile]: just a comment to test

File: Gemfile [hunk 4]
```diff
@@ -14,6 +14,7 @@ gem 'sassc-rails'
 gem 'simple_form'
```
Comment (lauwhite) [inline Gemfile]: comment on line 14

File: Gemfile.lock [hunk 5]
```diff
@@ -53,6 +54,12 @@ GEM
     coderay (1.1.2)
     concurrent-ruby (1.1.5)
     crass (1.0.4)
+    devise (4.6.2)
+      bcrypt (~> 3.0)
+      orm_adapter (~> 0.1)
```
Comment (bkneussl) [inline Gemfile.lock]: oh nooo me no like

Comment (annelaws) [reply Gemfile.lock]: i don't care

**Pull request merged**